### PR TITLE
Collect VA medical centers on pension form

### PR DESCRIPTION
**Pull request by jbalboni**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3795

Collects just the name/location for VA medical centers where the Vet has been treated.

screen shot 2017-07-11 at 10 17 51 am

### CODE REVIEW

File: src/js/pensions/config/form.js [hunk 1]
```diff
@@ -444,14 +447,27 @@ const formConfig = {
...
+            vamcTreatmentCenters: {
+              'ui:description': 'Please enter all VA medical centers where you have received treatment',
+              'ui:options': {
+                viewField: MedicalCenterField,
+                itemName: 'Medical Center',
+                expandUnder: 'view:hasVisitedVAMC'
+              },
+              items: {
+                location: {
+                  'ui:title': 'Name and location of VA medical center'
```
Comment (cehsu) [inline src/js/pensions/config/form.js]: Should we include the parenthetical about the location here as well (city, state)?

**Review by cehsu — APPROVED** [review]
Looks good to me. Had one question/comment.

File: src/js/pensions/config/form.js (hunk 1 above)
Comment (jbalboni) [inline src/js/pensions/config/form.js]: Couldn't hurt.

File: test/pensions/config/disabilityHistory.unit.spec.jsx [hunk 2]
```diff
@@ -108,15 +107,36 @@ describe('Pensions disability history', () => {
         value: '2003'
       }
     });
-    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_hasVisitedVAMCYes'), {
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_view\\:hasVisitedVAMCNo'), {
```
Comment (cvalarida) [inline test/pensions/config/disabilityHistory.unit.spec.jsx]: `formDOM` has a `selectYesNo()` for this.

**Review by cvalarida — APPROVED** [review]

**Pull request merged**